app/assets/stylesheets/admin.scss

```diff
@@ -11,6 +11,58 @@ label {
   margin-bottom: 0 !important;
 }
 
+h1 {
```

Comment:
I do agree with your overall reasoning that we should be using crayons-style everywhere in should not override h1-h6 tags in admin.scss file.

But this PR is purely focused on removing bootstrap only and as a I result I have not added those classes to h1-h6 tags.

Once we are done with removing bootstrap I think it would be much more clean that we then remove these h1-h6 tags too and instead use appropriate crayons-class.

My reason is that if we apply those changes directly in this PR then there might be introduction of some new crayon-classes and as a result might create confusion if someone revisits this PR in future.
Also, wanted to keep this PR simple, because if something breaks after merging it because of bootstrap removal, it would be easy to revert this PR and fix those issues.


Still, do let me know if apply those changes directly in this PR makes more sense, would be happy to fix them directly in here.

Reply:
Based on some more investigation it seems that we already have the `crayons-title` and `crayons-subtitle-x` classes on most of the headings which was a nice surprise. 

However, I think what you've outlined above is fine, let's tackle this in a follow up PR instead  ✨

Reply:
Great. Thanks.